analyzers/src/SonarAnalyzer.CSharp/Constants/SyntaxConstants.cs

```diff
@@ -23,5 +23,8 @@ namespace SonarAnalyzer.Constants
     public static class SyntaxConstants
```

Comment:
Isn't SyntaxLiteral(s) a better name?

Reply:
Thanks for the suggestion, we will consider it. However, it will not be changed anyway in this PR as we want to merge it as soon as possible to release in time.

Reply:
@csaba-sagi-sonarsource I fully agree. But as it was mentioned here, so I thought is was worth making the point.